test/test-platform-output.c

```diff
@@ -88,6 +88,28 @@ TEST_IMPL(platform_output) {
   printf("  maximum resident set size: %llu\n",
          (unsigned long long) rusage.ru_maxrss);
 
+#ifndef RUSAGE_THREAD
+  err = uv_getrusage_thread(&rusage);
+  ASSERT(err = UV_ENOTSUP);
+#else
+  err = uv_getrusage_thread(&rusage);
+  ASSERT(err == 0);
+  ASSERT(rusage.ru_utime.tv_sec >= 0);
+  ASSERT(rusage.ru_utime.tv_usec >= 0);
+  ASSERT(rusage.ru_stime.tv_sec >= 0);
+  ASSERT(rusage.ru_stime.tv_usec >= 0);
```

Comment:
Could the new assert macros (`ASSERT_EQ`, `ASSERT_UINT64_GE`, etc) be used here?